### ISSUE TITLE
Setup user interface for upload

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#14]
## User story
1. As a user
2. I want a web application with a UI for uploading files
3. So that I can navigate the upload functionality easily

## Acceptance criteria
* A UI where users have a web app that shows them a place to upload files.
* -- Mockup??

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
